### PR TITLE
[WIP] - SSL test case

### PR DESCRIPTION
**Pull request by dgaikwad**
Adding test to check is ssl overwritting or not after connect with freeipa

### PRT Run

{{pytest: cfme/tests/cli/test_appliance_console.py::test_appliance_overwrite_ssl_ipa  -v}}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 05d3f30f59dc9fd2c62262d6544e44dc0741f38d

The *local* fixture **`freeipa_provider`** is used in the following files:
- **cfme/tests/cli/test_appliance_console.py**
    - *test_appliance_overwrite_ssl_ipa*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit a8142eb225d38b5a6b87f51166b2c0dd06742c12

The *local* fixture **`freeipa_provider`** is used in the following files:
- **cfme/tests/cli/test_appliance_console.py**
    - *test_appliance_overwrite_ssl_ipa*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e21478c386d433bf951707c997f49fc6d9950720:

`cfme/tests/cli/test_appliance_console.py`:
- :bangbang: Line 82:1: **E303** *too many blank lines (3)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
I detected some fixture changes in commit e21478c386d433bf951707c997f49fc6d9950720

The *local* fixture **`freeipa_provider`** is used in the following files:
- **cfme/tests/cli/test_appliance_console.py**
    - *test_appliance_overwrite_ssl_ipa*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 84fb08318c4733517dbbcfd72871c14e112c5007

The *local* fixture **`freeipa_provider`** is used in the following files:
- **cfme/tests/cli/test_appliance_console.py**
    - *test_appliance_overwrite_ssl_ipa*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
Lint report for commit 84fb08318c4733517dbbcfd72871c14e112c5007:

`cfme/tests/cli/test_appliance_console.py`:
- :bangbang: Line 82:1: **E303** *too many blank lines (3)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 8a26c406b83417197ea311e25bb0e56762e51dc9

The *local* fixture **`freeipa_provider`** is used in the following files:
- **cfme/tests/cli/test_appliance_console.py**
    - *test_appliance_overwrite_ssl_ipa*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/cli/test_appliance_console.py [hunk 1]
```diff
@@ -1652,3 +1660,68 @@ def test_appliance_console_vmdb_log_negative(temp_appliance_preconfig_funcscope)
...
+    Bugzilla:
+        1670138
+
+    Polarion:
+        assignee: dgaikwad
+        casecomponent: Appliance
+        caseimportance: high
+        initialEstimate: 1/8h
+        testSteps:
+            1. appliance_console_cli --region=1 --internal --username=root --password=redhat
```
Comment (john-dupuy) [inline cfme/tests/cli/test_appliance_console.py]: **required**, should not put creds here.

File: cfme/tests/cli/test_appliance_console.py [hunk 2]
```diff
@@ -1652,3 +1660,68 @@ def test_appliance_console_vmdb_log_negative(temp_appliance_preconfig_funcscope)
     assert "Validated failed with error" in result[-1], "Database Configuration is not failed"
 
     ap_console_tail.validate(wait="60s")
+
+
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(automates=[1670138])
+@pytest.mark.meta(blockers=[BZ(1670138, forced_streams=["5.10"])])
+def test_appliance_overwrite_ssl_ipa(unconfigured_appliance, freeipa_provider):
+    """test to check ssl modifying or not  when it install from freeipa server
```
Comment (john-dupuy) [inline cfme/tests/cli/test_appliance_console.py]:
```suggestion
    """Test to check whether appliance ssl is modified when imported from freeipa server.
```

File: cfme/tests/cli/test_appliance_console.py [hunk 3]
```diff
@@ -1652,3 +1660,68 @@ def test_appliance_console_vmdb_log_negative(temp_appliance_preconfig_funcscope)
...
+
+    result = appliance.ssh_client.run_command(server_cer_command)
+    assert result.success
+    server_cer_op = result.output
+
+    result = appliance.ssh_client.run_command(server_cer_key_cmd)
+    assert result.success
+    server_cer_key_op = result.output
+
+    command = ("appliance_console_cli --region=1 --internal --username=root --password=redhat"
```
Comment (john-dupuy) [inline cfme/tests/cli/test_appliance_console.py]: same comment here about the passwds

**Pull request closed without merging**